### PR TITLE
Some basic v2 scenarios

### PR DESCRIPTION
**Pull request by jflatow**
This change isn't specific to the DSR branch, but running against that migration looks like there might be some issues redeeming after minting to look into...

### CODE REVIEW

**Comment by kevincheng96** [comment]
Neat! Is the plan to merge this into main or keep it as a branch?

**Comment by jflatow** [comment]
> Neat! Is the plan to merge this into main or keep it as a branch?

I think we can merge and store w/ tests skipped on `main`, but probably don't need to merge into this branch (thats why its a draft)?

File: scenario/V2Scenario.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+// note: meant for sanity checking v2 proposals, can normally be set to skip
+//  enable to check specific v2 actions with addresses hard-coded by network
+//  addresses to check and markets will need to be updated between runs
+
+scenario.skip(
+  'Compound v2 > allows a user to repay, borrow, repay cETH',
+  {},
+  async ({ actors }, context, world) => {
+    const { albert, betty } = actors; albert; betty;
```
Comment (scott-silver) [inline scenario/V2Scenario.ts:13]: just curious, what does appending `albert; better;` to this line do?

isn't this the same as just:

```
const { albert, betty } = actors;
```

Comment (jflatow) [reply scenario/V2Scenario.ts:13]: Whoops good catch!

**Review by scott-silver — APPROVED** [review]
🔧

**Review by kevincheng96 — APPROVED** [review]
A couple lint issues, but otherwise LGTM!

File: scenario/V2Scenario.ts (hunk 1 above)
Comment (jflatow) [reply scenario/V2Scenario.ts]: Actually these were to quiet linter; but deleted the actors altogether now

**Pull request merged**